### DISCUSSION TITLE
How to replicate the schema prism model in the database

### DISCUSSION THREAD

**Discussion by felinto-dev** [prisma/prisma#2306] [Q&A] [answered]
Hello,

I learned prism 1 and now I'm trying to learn prism 2.

In prism 1, after the schema was generated I could simply run the prisma deploy command and the schema was replicated in the database.

Isn't there something similar in prisma 2?

**Comment by ryands17** [comment, marked as answer]
Hi @felinto-dev :wave: 

Prisma2 is different in a way from Prisma1 that it doesn't require any additional server to deploy to,
It is just a DB access layer that helps you query your database.

You can get started with the examples [here](https://github.com/prisma/prisma-examples) and also learn more about getting started with Prisma 2 [here](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch-prisma-migrate-typescript-postgres)

You can use `prisma migrate`. It is similar to `prisma deploy` in the sense that it creates all migrations against your database.

For e.g.

1. Create your model in the `schema.prisma`

```prisma
model Post {
  id        Int      @default(autoincrement()) @id
  createdAt DateTime @default(now())
  title     String
  content   String?
  published Boolean  @default(false)
  author    User     @relation(fields: [authorId], references: [id])
  authorId  Int
}

model User {
  id      Int      @default(autoincrement()) @id
  email   String   @unique
  name    String?
  posts   Post[]
}
```

2. Run `prisma migrate save --experimental`
This will save the migrations in a folder.

3. Run `prisma migrate up --experimental`
This will run the migrations against your DB similar to `prisma deploy` now you will be able to view those fields in the database.

Lastly, `prisma generate` will generate the all your required queries which you can use in your code!

Let me know if this works for you :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
